PersonalizedCampus/screens/tourCreationScreen.js

```diff
@@ -53,11 +53,12 @@ const TourCreationScreen = props => {
                 firebase.database().ref('/tours/' + userID )
                 .push({
                     tourName: data.title,
-                    public_private: 'private',
                     owner: userID, // Redundant
                     createdAt: Date.now(),
                     lastModified: Date.now(),
                     anchor: data.anchor,
+                    private: true,
```

Comment:
@Doxie300 I changed your public_private string field to a boolean, I don't think it really matters but figured it might be a better practice. Feel free to change to your preference.